VirtusizeCore/Sources/API/APIService.swift

```diff
@@ -44,6 +44,9 @@ open class APIService {
 		delegateQueue: nil
 	)
 
+    /// Cached Store ID
+    public static var currentStoreId: Int?
```

Comment:
It's not a good place to cache the store ID, but based on the current architecture, it's the easiest solution to cache the store ID.

Reply:
Understood.